### PR TITLE
add basic unit test

### PR DESCRIPTION
**Pull request by jellegerbrandy**
* define `npm run test:unit`
* write a unit test for formatToken
* make sure that formatToken rounds down (to address https://daostack.tpondemand.com/RestUI/Board.aspx#page=profile&boardPopup=userstory/1258/silent)
* make the App.spec js test work

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,35 +16,40 @@
...
       "tsx",
       "js",
       "jsx",
-      "json"
+      "json",
+      "node"
     ],
     "moduleNameMapper": {
-      "\\.(scss|css|less)$": "identity-obj-proxy",
+      "\\.(scss|css|less|svg|png|jpg)$": "identity-obj-proxy",
```
Comment (tibetsprague) [inline package.json]: what is this for?

File: src/lib/util.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ import {
   IProposalState,
   IRewardState,
   ISchemeState } from "@daostack/client";
-import { GenericSchemeRegistry } from "genericSchemeRegistry";
+import { GenericSchemeRegistry } from "../genericSchemeRegistry";
```
Comment (tibetsprague) [inline src/lib/util.ts]: isn't this fixed by the change you made in package.json?

File: package.json (hunk 1 above)
Comment (jellegerbrandy) [reply package.json]: this is for jest to understand what to do with svg  png jpg files (namely: nothing at all)

File: src/lib/util.ts (hunk 2 above)
Comment (jellegerbrandy) [reply src/lib/util.ts]: ah yeah that's from before the fix

**Pull request merged**